### PR TITLE
Don't store multiple follow request from a single person

### PR DESCRIPTION
**Pull request by annando**
We are currently flooding the net with follow requests for items. As a first step we now only store them once per contact and not our own. This should also prevent false notifications.

### CODE REVIEW

File: src/Model/User.php [hunk 1]
```diff
@@ -99,6 +99,20 @@ public static function getById($uid)
 		return DBA::selectFirst('user', [], ['uid' => $uid]);
 	}
 
+	/**
+	 * @param  integer       $uid
+	 * @return array|boolean User record if it exists, false otherwise
```
Comment (MrPetovan) [inline src/Model/User.php]: Does it return a `User` record or a public Contact Id?

Comment (annando) [reply src/Model/User.php]: It does return some contact stuff - but the input value is the user id. I had been unsure where to place it. In the Contact class it had to named differently as well, I think.

Comment (MrPetovan) [reply src/Model/User.php]: It probably should by in Contact as `getPublicIdByUserId` and have an `int` return value.

Comment (annando) [reply src/Model/User.php]: Moved

**Pull request merged**